proposals/NNNN-unlock-existential-types-for-all-protocols.md

```diff
@@ -217,7 +217,7 @@ protocol Q: P where B == G<A> {}
 ```
 Notice how the associated type that would preclude a call to `method` on a value of type `Q` is actually `A`, not `B` as the result type may suggest, due to the same-type constraint on the protocol.
 
-#### Non-conformable Existentials
+#### Non-Conformable Existentials
```

Comment:
By contrast with "type-erasing", "non" is merely a prefix, not a standalone word, so the second element was correct to be in lower case.